ascii/valid_print_asm.go

```diff
@@ -0,0 +1,117 @@
+// +build ignore
```

Comment:
This was `valid_asm.go`, containing the `validPrint16` function, moved to `valid_print_asm.go`.